components/EnoughToBuy.tsx

```diff
@@ -1,16 +1,63 @@
+import { useState, useEffect } from "react";
+import { useAccount } from "wagmi";
+
 import { getImpactItems, isLiftPersonOutOfPovertyImpactItem } from "@/utils";
 
+import type { GetImpactItem } from "@/utils";
+
 type Props = {
   yearlyYield: number;
 };
 export default function EnoughToBuy({ yearlyYield }: Props) {
-  const yearlyImpactItem = getImpactItems(yearlyYield)[0];
+  const yearlyImpactItems = getImpactItems(yearlyYield);
   const enoughToLiftPersonOutOfPoverty =
-    yearlyImpactItem && isLiftPersonOutOfPovertyImpactItem(yearlyImpactItem);
+    yearlyImpactItems[0] &&
+    isLiftPersonOutOfPovertyImpactItem(yearlyImpactItems[0]);
+  const impactItemOffset = (yearlyImpactItems.length - 1) * 24;
+  const [style, setStyle] = useState({
+    transform: `translateY(-${impactItemOffset}px)`,
+    opacity: "0",
+    transition: "all 1s",
+  });
+  const { isConnected } = useAccount();
+  useEffect(() => {
+    if (isConnected) {
+      const fadeinTimer = setTimeout(() => {
```

Comment:
Perhaps [Framer](https://www.framer.com/motion/) that we introduced recently would be a cleaner approach? :sweat_smile:

Reply:
sure if that's easier to maintain